### DISCUSSION TITLE
Android high refresh rate screens

### DISCUSSION THREAD

**Discussion by DelinWorks** [axmolengine/axmol#812] [Q&A] [answered]
Hi! I would like to know how to set the refresh rate dynamically on android devices with 120hz screens based on the device's screen refresh rate (i.e. make the game choose either 60fps or 120fps based on user screen preference) something like glfwGetVideoMode in glfw. 
setting fps count like this `director->setAnimationInterval(1.0f / 120);` works but isn't viable, or so I think it is.. correct me If I'm wrong.
Thanks!

**Comment by aismann** [comment]
@DelinWorks 
Any news? An answer found?

**Comment by DelinWorks** [comment, marked as answer]
turns out the solution is vendor specific for each android device so sticking to just infinite fps and letting the system cap the fps is the appropriate solution
